MinecraftClient/Commands/Inventory.cs

```diff
@@ -110,6 +110,25 @@ public override string Run(McTcpClient handler, string command, Dictionary<strin
                                     }
                                 }
                                 else return CMDDesc;
+                            case "creative":
+                                if (args.Length >= 3)
+                                {
+                                    int slot = int.Parse(args[2]);
+                                    ItemType ItemType = (ItemType)int.Parse(args[3]);
```

Comment:
Maybe [Enum.Parse](https://docs.microsoft.com/en-us/dotnet/api/system.enum.parse?view=netcore-3.1) would help converting string representation of object name e.g `Stone` into the ItemType value for stone? Of course you can keep support for IDs as long as they don't get randomized by Mojang between Minecraft versions.